common/v2/components/AccountList.tsx

```diff
@@ -15,21 +15,57 @@ import {
 import { DashboardPanel } from './DashboardPanel';
 import './AccountList.scss';
 
+const Label = styled.span`
+  display: flex;
+  align-items: center;
+  p {
+    margin-right: 27px;
+  }
+`;
+
+interface iFavoriteProps {
+  favorited: boolean;
+}
+
+const FavoriteButton = styled(Button)`
+  span {
+    span {
+      svg {
+        path {
+          fill: ${(props: iFavoriteProps) => (props.favorited ? 'rgb(255, 209, 102)' : 'white')};
+          stroke: ${(props: iFavoriteProps) =>
+            props.favorited ? 'rgb(255, 209, 102)' : '#7b8695'};
+        }
+      }
+    }
+  }
+  align-self: flex-start;
+  margin-left: 1em;
+`;
+
 const DeleteButton = styled(Button)`
   align-self: flex-start;
   margin-left: 1em;
 `;
 
+const TableContainer = styled.div`
+  display: block;
+  max-height: 394px;
+  overflow: auto;
+`;
+
 type DeleteAccount = (uuid: string) => void;
+type UpdateAccount = (uuid: string, accountData: ExtendedAccount) => void;
 interface AccountListProps {
   className?: string;
   currentsOnly?: boolean;
+  deletable?: boolean;
```

Comment:
In which case do we choose if we want to delete an account or not?

Reply:
I believe it depends on where the account list is being displayed. e.g. we could only make accounts deletable from the settings page, not the dashboard, or have a toggle somewhere that reveals a delete button to keep the UI clean.

Reply:
Ah, ok!
Settings vs Dashboard seems like a simple option to start with.

Reply:
Yep. I already set `deletable` to true on the Settings page